.github/workflows/cppcheck.yml

```diff
@@ -0,0 +1,20 @@
+name: cppcheck-action-test
+on: [push]
+
+jobs:
+  build:
+    name: cppcheck-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: cppcheck
+        uses: deep5050/cppcheck-action@main
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN}}
```

Comment:
I was following their [default configuration](https://github.com/marketplace/actions/cppcheck-action#default-configuration).

> Additionally, is there any benefit to using that action instead of simply writing one ourselves? 


I did not know that we could write our own github action. I will make the necessary change.